chapters/stack/functions/guides/hello_world/support/hello_world.asm

```diff
@@ -0,0 +1,19 @@
+%include "../utils/printf32.asm"
+
+section .data
+    msg db 'Hello, world!', 0
+
+section .text
+
+extern puts
```

Comment:
```suggestion
extern puts ; we need to mark "puts()" as an external function, not one found in this file
```

I'd add even more details than what I sugested. Make sure that the students understan what `extern` is. Perhaps even add a section about it in the main content?